README.md

```diff
@@ -38,9 +38,9 @@ Bitte benutze [![Authentication](https://img.shields.io/badge/Auth-OAuth2-green)
 
 ## Anwendungsfälle der API
 
-- [Stammdaten eines Partners auslesen](docs/Partner_auslesen.md)
-- [neuen Benutzer anlegen](docs/Partner_anlegen.md)
-- [Zugang anlegen](docs/Zugang.md)
-- [Partnerkennzeichen auslesen](docs/Partnerkennzeichen_auslesen.md)
-- [Rechte eines Benutzer auslesen](docs/Rechte_auslesen.md)
-- [Partner aktualisieren](docs/Partner_aktualsieren.md)
+- [Stammdaten eines Partners auslesen](https://docs.api.europace.de/baufinanzierung/partner/partner-api/partner_auslesen/)
```

Comment:
find ich nicht gut. das ist für rate und baufi gleichermaßen und hier ist nur ein link auf Baufi. Das ist für die Kunden von Kreditsmart nicht verständlich

Reply:
Ja, hier rächt sich die Struktur und das dass PM nicht in einem eigenen Verzeichnis in der API-Site liegt. Gerne suche ich später nach einer Lösung dafür.

Reply:
@MikeKrueger75  wieso können wir es denn nciht so lassen wie es vorher war und die Beispiele hier lassen bis wir eine Lösung gefunden haben?